xml/admin_ceph_upgrade.xml

```diff
@@ -781,11 +781,11 @@ ID   CLASS  WEIGHT   TYPE NAME       STATUS  REWEIGHT  PRI-AFF
       to specify the image to use on each command, for example:
      </para>
 <screen>
-&prompt.cephuser;cephadm --image 192.168.121.1:5000/ses/7/prometheus/prometheus-server:2.27.1 \
+&prompt.cephuser;cephadm --image 192.168.121.1:5000/ses/7.1/prometheus/prometheus-server:2.27.1 \
```

Comment:
```suggestion
&prompt.cephuser;cephadm --image 192.168.121.1:5000/ses/7.1/prometheus/prometheus-server:2.32.1 \
```

we've bumped the prometheus-server version (https://github.com/SUSE/ceph/pull/455)